docs/doc/03-develop/01-Python/00-installing.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Installing the Python Connector
```

Comment:
It's better to combine them all to a single page, since each peice is not too long: https://databend.rs/doc/develop/, this is easy for share.
For the chapter, we can add subtitles for them.

Reply:
Make sense.